### ISSUE TITLE
Set site html lang from site settings?

### ISSUE THREAD

**Issue by CrispinF** [cloudscribe/cloudscribe#569]
A site will fail accessibility checks if the html lang attribute is missing (as it is by default in the supplied theme layout files).  Could this be set from Site Settings?
This would help where two tenant sites in different languages share the same skin, but where this attribute should be switched. 
Or is there a good way of doing this already?

**Comment by joeaudette** [comment]
Probably we should add a variable in the layout and default it to en-US, but override it if sitesettings has a value for forced culture.

**Comment by CrispinF** [comment]
Possibly just "en" as default?

**Comment by joeaudette** [comment]
sure

**Comment by joeaudette** [comment]
this is now implemented, nugets and new project template published
